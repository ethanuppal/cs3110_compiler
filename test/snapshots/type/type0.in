func main() {
    print true
    return
}
