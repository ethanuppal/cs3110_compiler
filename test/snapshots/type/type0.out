func main() -> Unit {
    print true
    return
}
